robolectric/src/test/java/org/robolectric/shadows/ShadowSettingsTest.java

```diff
@@ -60,6 +60,19 @@ public void testSystemGetString() throws Exception {
   }
 
   @Test
+  @Config(sdk = {
+          Build.VERSION_CODES.JELLY_BEAN_MR1,
+          Build.VERSION_CODES.JELLY_BEAN_MR2,
+          Build.VERSION_CODES.KITKAT,
+          Build.VERSION_CODES.LOLLIPOP,
+          Build.VERSION_CODES.LOLLIPOP_MR1,
+          Build.VERSION_CODES.M })
+  public void testGetStringForUser() throws Exception {
+    Settings.System.getStringForUser(contentResolver, "someUser", -1);
+    assertThat(true);
```

Comment:
assertThat true?

I don't think you should write a test for this code since it's not part of the API. Rather you should write a test for DateUtil.formatDateTime() which exposed this issue on M.